node/tests/schemaStruct.go

```diff
@@ -43,8 +43,8 @@ func SchemaTestRequiredFields(t *testing.T, engine Engine) {
 		ma, _ := nb.BeginMap(0)
 		err := ma.Finish()
 
-		Wish(t, err, ShouldBeSameTypeAs, schema.ErrMissingRequiredField{})
-		Wish(t, err.Error(), ShouldEqual, `missing required fields: a,b`)
+		qt.Check(t, err, qt.ErrorAs, &schema.ErrMissingRequiredField{})
```

Comment:
Please pay attention to this modification; I'd love to make sure the changes are equivalent to what `ShouldBeSameTypeAs` is meant to do. 🍻

Reply:
I think the semantics here are slightly different, as errors.As unwraps error values - but that's fine, because the new semantics look better. The tests should enforce `errors.As`, not the error's top-level type.